src/test/cs/Infinispan/HotRod/NearCacheEvictionTest.cs

```diff
@@ -55,6 +55,7 @@ public void PutGetTest()
             Assert.AreEqual(stats2.GetIntStatistic("misses"), stats3.GetIntStatistic("misses"));
             Assert.AreEqual(stats2.GetIntStatistic("hits") + 3, stats3.GetIntStatistic("hits"));
             cache.Clear();
+	    remoteManager.Stop();
```

Comment:
Is the manager stopped when the test fails somewhere in the middle? I think this could cause trouble for the rest of the test suite when the manager is not stopped, right?